pkgs/by-name/sc/screenconfig/package.nix

```
@@ -0,0 +1,37 @@
+{
+  lib,
+  python3,
+  fetchFromGitHub,
+  xrandr,
+  srandrd,
+  feh,
+}:
+
+python3.pkgs.buildPythonPackage rec {
+  pname = "screenconfig";
+  version = "0.1.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "jceb";
+    repo = "screenconfig";
+    tag = "v${version}";
+    hash = "sha256-X1Mz8UbOOW/4LM9IZoG/kbwv2G0EppTsacKapQMChkc=";
+  };
+  build-system = [ python3.pkgs.setuptools ];
+  propagatedBuildInputs = [
+    python3.pkgs.toml
+    xrandr
+    srandrd
+    feh
+  ];
```

Separate python & other deps
```suggestion
  dependencies = with python3.pkgs; [
    toml
  ];
  
  buildInputs = [
    xrandr
    srandrd
    feh
  ];
```